### PR TITLE
Enhance Portal Setup

### PR DESCRIPTION
**Pull request by tujoworker**
This PR is made with Yarn PnP in mind – and it works fine, except that both VSCode integrations for ESLint and Prettier do not work properly with PnP. While there are other extensions which do, I think we go for the time being with node_modules.

All other enhancements are made with separate commits.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "search.exclude": {
+    "**/public": true,
+    "**/.cache": true,
+    "**/.yarn": true,
+    "**/.pnp.*": true
+  }
+}
```
Comment (dinarosv) [inline .vscode/settings.json:8]: These files are commited on purpose, right?

File: packages/dnb-design-system-portal/.ncurc.json [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+    "eslint",
+    "ora",
+    "react-router-dom",
+    "stylelint",
+    "stylelint-config-prettier",
+    "stylelint-config-recommended",
+    "stylelint-config-standard",
+    "stylelint-scss"
+  ]
 }
```
Comment (dinarosv) [inline packages/dnb-design-system-portal/.ncurc.json:3]: What does this file do?

File: packages/dnb-design-system-portal/src/docs/design-system/changelog.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ draft: true
 order: 1
 ---
 
-import ChangeLog from 'Root/../../CHANGELOG.md'
-import ReleasesInfo from 'Pages/design-system/changelog/info-about-releases'
```
Comment (dinarosv) [inline packages/dnb-design-system-portal/src/docs/design-system/changelog.md:9]: Great!

**Review by dinarosv — APPROVED** [review]
Great work!👏 And great amount of work 😄

File: .vscode/settings.json (hunk 1 above)
Comment (tujoworker) [reply .vscode/settings.json:8]: the .yarn folder we have, the .pnp file we dont. But its just to let vscode know, what to ignore by default when you as a dev search the repo.

File: packages/dnb-design-system-portal/.ncurc.json (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/.ncurc.json:3]: I may mention these in the docs! Even its not the purpose that other devs do update the deps. But they are a run command config for [ncu](https://www.npmjs.com/package/npm-check-updates).

Comment (dinarosv) [reply packages/dnb-design-system-portal/.ncurc.json:3]: Aaah, yes might be good to mention in the docs 🙌

**Pull request merged**